lib/win_checker.rb

```diff
@@ -18,9 +19,7 @@ def has_player_won?(token)
   private
 
   def get_full_sets(side_length)
```

Comment:
This method doesn't use the argument passed in.

Reply:
Removed the side_length argument.